### ISSUE TITLE
gnuace MPC template can't handle include/lib paths with spaces

### ISSUE THREAD

**Issue by jwillemsen** [DOCGroup/ACE_TAO#2281]
When using cross compilation from Windows and using some 3rd party C++ headers installed in a path with a space I found that the gnuace template can't handle that, include/lib paths aren't quoted when there is a space in the path. For example I see on the commandline of g++ `-IC:/Program Files/foo//include`

**Comment by mitza-oci** [comment]
Try something like:

`CPPFLAGS     +=<%foreach(includes)%> -I$(call ADD_QUOTE, <%include%>)<%endfor%>`

Or you can use a SUBST path on Windows to set up the paths to not have spaces.

**Comment by jwillemsen** [comment]
Yes, should be something like that, stored it here as a todo, much more higher priority work at this moment
